ipamapi/contract.go

```diff
@@ -42,59 +51,22 @@ type AddressSpace string
 // Common key is a addressspace
 type Config interface {
```

Comment:
When I said Driver in 
> I think these Config APIs must be driver specific & not to be prescribing by the core.

I meant IPAM plugin/driver. Not the Network plumbing driver.